examples/sample-app/cleanup.sh

```diff
@@ -3,8 +3,13 @@
 echo "Killing openshift all-in-one server ..."
 pkill -x openshift
 
+echo "Stopping all k8s docker containers on host ..."
+docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
```

Comment:
This prints the name of the container, not the name of the image the container is running. Container names are unique, so there is not an issue with collisions.